test/test_cuda.py

```diff
@@ -4659,9 +4441,212 @@ def test_no_triton_on_import(self):
+                    for j, p in enumerate(params_graphed):
+                        p.grad = grads[i + steps_warmup][j]
+                    opt.step()
+
+            for p_control, p_graphed in zip(params_control, params_graphed):
+                self.assertEqual(p_control, p_graphed)
+
+    @onlyCUDA
+    @unittest.skipIf(
+        not TEST_CUDA_GRAPH, "CUDA >= 11.0 or ROCM >=5.3 required for graphs"
+    )
+    @optims(
+        [
+            optim
+            for optim in optim_db
+            if optim.optim_cls
+            in [
+                torch.optim.NAdam,
+                torch.optim.RAdam,
+                torch.optim.Rprop,
+                torch.optim.Adam,
+                torch.optim.AdamW,
+                torch.optim.Adamax,
+                torch.optim.ASGD,
+                torch.optim.Adadelta,
+                torch.optim.RMSprop,
+            ]
+        ],
+        dtypes=[torch.float32],
+    )
+    def test_graph_optims(self, device, dtype, optim_info):
+        optim_cls = optim_info.optim_cls
+        all_optim_inputs = _get_optim_inputs_including_global_cliquey_kwargs(
+            device, dtype, optim_info, skip=("differentiable",)
+        )
+        has_betas = any(
+            "betas" in error_inp.optimizer_error_input.kwargs
+            for error_inp in optim_info.optim_error_inputs_func(
+                device="cpu", dtype=dtype
+            )
+        )
+        for optim_input in all_optim_inputs:
+            kwargs = optim_input.kwargs
+            if "lr" in kwargs:
+                del kwargs["lr"]
+            kwargs["lr"] = 0.1
+            if has_betas and optim_cls != torch.optim.Adamax:
+                kwargs["betas"] = (0.8, 0.7)
+            with self.subTest(optimizer_ctor=optim_cls, kwargs=kwargs):
+                self._test_graphed_optims(3, 2, optim_cls, kwargs)
+
+    @onlyCUDA
+    @unittest.skipIf(
+        not TEST_CUDA_GRAPH, "CUDA >= 11.0 or ROCM >= 5.3 required for graphs"
+    )
+    @optims(
+        [
+            optim
+            for optim in optim_db
+            if optim.optim_cls in [torch.optim.Adam, torch.optim.AdamW, torch.optim.SGD]
+        ]
+    )
+    def test_graph_scaling_fused_optimizers(self, device, dtype, optim_info):
+        optim_cls = optim_info.optim_cls
+
+        steps_warmup = 3
+        steps_train = 2
+
+        optim_inputs = optim_info.optim_inputs_func(device=device)
+        has_betas = any(
+            "betas" in error_inp.optimizer_error_input.kwargs
+            for error_inp in optim_info.optim_error_inputs_func(
+                device="cpu", dtype=dtype
+            )
+        )
+
+        for optim_input in optim_inputs:
+            kwargs = optim_input.kwargs
+            kwargs["fused"] = True
+            if "lr" in kwargs:
+                del kwargs["lr"]
+            kwargs["lr"] = 0.1
+            if has_betas:
+                kwargs["betas"] = (0.8, 0.7)
```

Comment:
no need

Reply:
If you want to override betas, then just do a 
```suggestion
            if "betas" in kwargs:
                kwargs["betas"] = (0.8, 0.7)
```

Reply:
Here, for torch.optim.Adam and torch.optim.AdamW none of the kwargs have betas, but in the original test, all of the tests have these specific betas, so I thought it might be better to inject the betas into the kwargs in the function rather than adding these betas to the configs since then the configs might not be as versatile.